integration_tests/deploy/test_deploy.sh

```diff
@@ -22,58 +22,119 @@ source integration_tests/cli/common.sh
+
+    echo "$node: upload release artifacts"
+    docker cp deployment/dcld.service "$node":"$DCL_USER_HOME"
+    docker cp deployment/scripts/run_dcl_node "$node":"$DCL_USER_HOME"
+    docker cp deployment/scripts/test_peers_conn "$node":"$DCL_USER_HOME"
+
+    echo "$node: init"
+    docker_exec "$node" bash -c "dcld init $node --chain-id $CHAIN_ID 2>init.stderr"
+
+    echo "$node: Create NodeAdmin and Trustee keys"
+    docker_exec "$node" dcld config keyring-backend test
+    docker_exec "$node" dcld keys add "${node}_admin"
+    docker_exec "$node" dcld keys add "${node}_tr"
+done
+
+echo "Generating persistent peers list"
+GVN_ID="$(docker_exec "$GVN_NAME" grep -o 'node_id.*$' init.stderr  | awk -F'"' '{print $3}')"
+VN_ID="$(docker_exec "$VN_NAME" grep -o 'node_id.*$' init.stderr  | awk -F'"' '{print $3}')"
+
+PERSISTENT_PEERS="$GVN_ID@$GVN_IP:26656,$VN_ID@$VN_IP:26656"
+
+for node in "$GVN_NAME" "$VN_NAME"; do
+    docker_exec "$node" bash -c "echo '$PERSISTENT_PEERS' >persistent_peers.txt"
+done
+
+
+test_divider
+
+echo "GENESIS NODE RUN"
+docker_exec -e KEYRING_BACKEND=test "$GVN_NAME"  \
+    ./run_dcl_node -t genesis -c "$CHAIN_ID" \
+    --gen-key-name "${GVN_NAME}_admin" --gen-key-name-trustee "${GVN_NAME}_tr" \
+    "$GVN_NAME"
+docker_exec "$GVN_NAME" systemctl status dcld
+
+# we shold be fine with shell limits here
+genesis_data="$(docker_exec "$GVN_NAME" cat "$GENESIS_FILE")"
+
+test_divider
+
+echo "VALIDATOR NODE RUN"
+
+echo "VN: Set genesis data"
+docker_exec "$VN_NAME" bash -c "echo '$genesis_data' >genesis.json"
+
+echo "VN: Verify connections to other nodes"
+docker_exec "$VN_NAME" ./test_peers_conn
+
+echo "VN: run"
+docker_exec -e KEYRING_BACKEND=test "$VN_NAME" ./run_dcl_node -c "$CHAIN_ID" "$VN_NAME"
+docker_exec "$VN_NAME" systemctl status dcld
+
+test_divider
+
+echo "VALIDATOR NODE REGISTRATION"
+
+echo "$GVN_NAME: approve \"$VN_NAME\" to validator set"
+vn_addr=$(docker_exec "$VN_NAME" dcld tendermint show-address)
+vn_pubkey=$(docker_exec "$VN_NAME" dcld tendermint show-validator)
+vn_admin_name="${VN_NAME}_admin"
+vn_admin_addr="$(docker_exec "$VN_NAME" dcld keys show "$vn_admin_name" -a)"
+vn_admin_pubkey="$(docker_exec "$VN_NAME" dcld keys show "$vn_admin_name" -p)"
+
+# ensure that the genesis node is ready
+wait_for_height 2 15 "tcp://$GVN_IP:26657"
+
+docker_exec "$GVN_NAME" dcld tx auth propose-add-account --address "$vn_admin_addr" --pubkey "$vn_admin_pubkey" --roles="NodeAdmin" --from "${GVN_NAME}_tr" --yes
+#dcld tx auth approve-add-account --address="$vn_admin_addr" --from alice --yes
+
+echo "$GVN_NAME: Add Node \"$VN_NAME\" to validator set"
+docker_exec "$VN_NAME" dcld tx validator add-node --pubkey="$vn_pubkey" --moniker="$VN_NAME" --from="$vn_admin_name" --yes
+
+echo "Check node \"$VN_NAME\" is in the validator set"
+result=$(docker_exec "$GVN_NAME" dcld query validator all-nodes)
+check_response "$result" "\"moniker\": \"$VN_NAME\""
+check_response "$result" "\"pubKey\":$vn_pubkey" raw
 
```

Comment:
It would be great to have some checks that nodes are catching up, participate in consensus (write and read requests can be sent).